### DISCUSSION TITLE
Sync error with extension v2023.12.0 on Edge

### DISCUSSION THREAD

**Discussion by koying** [dani-garcia/vaultwarden#4154] [Q&A] [answered]
### Subject of the issue
After the bitwarden extension was auot-updated to 2023.12.0 on Edge, synchornization with v1.30.1 fails.

### Deployment environment

* vaultwarden version:
1.30.1

* Install method:
Docker image: vaultwarden/server:1.30.1-alpine

* Clients used: 
Microsoft Edge 2023.12.0 has the issue (on 2 computers)
Other clients on 2023.10.2 work fine.

* Reverse proxy and version: 

* MySQL/MariaDB or PostgreSQL version: 

* Other relevant details:

### Steps to reproduce

![image](https://github.com/dani-garcia/vaultwarden/assets/164740/3f690f09-a150-40be-a151-158569601cf4)

### Expected behaviour

No error message and actual sync.

### Actual behaviour

Error and no sync

**Comment by BlackDex** [comment]
Could you share te logs of Vaultwarden please

**Comment by koying** [comment]
This is what I see in the docker log when I try a sync:
```
vaultwarden  | [2023-12-09 10:03:28.923][request][INFO] POST /identity/connect/token
vaultwarden  | [2023-12-09 10:03:28.927][response][INFO] (login) POST /identity/connect/token => 200 OK
vaultwarden  | [2023-12-09 10:03:29.314][request][INFO] GET /api/sync
vaultwarden  | [2023-12-09 10:03:29.384][response][INFO] (sync) GET /api/sync?<data..> => 200 OK
```

Basically the same as a successful one, it seems.

Is there a way to get some logs from the extension?

**Comment by BlackDex** [comment]
This seems to be an extension issue. I was able to reproduce this, but saw nothing strange server-side or client side.

I then fully logged-out of the extension, restarted the browser, logged-in again, and that solved my syncing issue.
I tested this with Chromium, but that is the same extension.

**Reply by masterwishx** [reply]
Also has has this issue, on mobile + extension Firefox and Chrome. 
Thanks fixed now.

**Comment by koying** [comment]
Ah, yes. Looks like logout/login makes sync work again, for now at least.
Will report back if the symptom happens again

**Comment by LunNova** [comment]
Might be this client issue https://github.com/bitwarden/clients/issues/7243

**Reply by BlackDex** [reply, marked as answer]
Which is solved by fully logout and login again.
